### PR TITLE
FP21-82-add-about-us-page about toolbox page

### PR DESCRIPTION
**Pull request by MJanOthman**
# Description

About toolbox page
real text and image added

# How to test?

run server and client 

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by hassou2n — APPROVED** [review]

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+}
+.container {
```
Comment (Divyajg) [inline packages/client/src/components/AboutToolbox/AboutToolbox.style.css]: Try naming this class more specific
For ex: about-toolbox-container

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  width: 90vw;
+  height: 40px;
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (Divyajg) [inline packages/client/src/components/AboutToolbox/AboutToolbox.style.css]: Does these styles `display: flex;
  justify-content: center;
  align-items: center;` make any difference?

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+.horizontal-solid-line {
+  width: 80%;
+  margin-right: 10%;
+  margin-left: 10%;
+  border-top: 2px;
+  border-color: #f5f5f5;
+  opacity: 0.5;
+}
+
+@media screen and (max-width: 650px) {
```
Comment (Divyajg) [inline packages/client/src/components/AboutToolbox/AboutToolbox.style.css:87]: The page beyond this width is acting weird. A couple of items are flowing out of the scope.

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  font-size: 1.6rem;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 65vw;
+  height: 100vh;
+}
+.about-toolbox-image {
+  margin: 7em;
+  width: 80%;
```
Comment (Divyajg) [inline packages/client/src/components/AboutToolbox/AboutToolbox.style.css]: With width, try `margin:auto;`

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css (hunk 3 above)
Comment (Divyajg) [reply packages/client/src/components/AboutToolbox/AboutToolbox.style.css:87]: Style the media screen with `min-width: 651px` for screens larger than 650px.

**Review by Divyajg — CHANGES_REQUESTED** [review]
KaosPilot-image is not responsive, fix that one.
The content is overflowing in several situations.
image
image

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css (hunk 4 above)
Comment (MJanOthman) [reply packages/client/src/components/AboutToolbox/AboutToolbox.style.css]: Some elements need different  values of margin.

**Review by Kristoffer88 — APPROVED** [review]

File: packages/client/src/components/AboutToolbox/AboutToolbox.style.css (hunk 3 above)
Comment (Divyajg) [reply packages/client/src/components/AboutToolbox/AboutToolbox.style.css:87]: You can use`(min-width: 651px)` and `(max-width: 650px)`, or `(max-width: 750px)` and `(min-width: 751px)` or `(max-width: 1230px)` and `(min-width: 1231px)` not all at once.
FYI: [Media queries width](https://www.freecodecamp.org/news/media-queries-width-ranges/#:~:text=Taking%20a%20look%20at%20the,extra%2Dsmall%20and%20small%20widths.)

**Review by Divyajg — APPROVED** [review]
Everything looks good except the toolbox-text is over-flowing out of the background image and I left you a comment about media queries.

**Pull request merged**